src/chocolatey/infrastructure.app/commands/ChocolateyInstallCommand.cs

```diff
@@ -20,21 +20,27 @@ namespace chocolatey.infrastructure.app.commands
     using System.Collections.Generic;
     using System.Linq;
     using attributes;
+    using chocolatey.infrastructure.app.nuget;
     using commandline;
     using configuration;
     using infrastructure.commands;
     using infrastructure.configuration;
     using logging;
+    using NuGet;
     using services;
 
     [CommandFor("install", "installs packages from various sources")]
     public class ChocolateyInstallCommand : ICommand
     {
         private readonly IChocolateyPackageService _packageService;
+        private readonly ChocolateyPinCommand _chocolateyPinCommand;
+        private readonly ILogger _nugetLogger;
 
-        public ChocolateyInstallCommand(IChocolateyPackageService packageService)
+        public ChocolateyInstallCommand(IChocolateyPackageService packageService, ChocolateyPinCommand chocolateyPinCommand, ILogger nugetLogger)

```

Comment:
This should not be adjusted if possible. It will require more review on this aspect.

Reply:
oh wait, nevermind.